cql3/Cql.g

```diff
@@ -1496,7 +1497,8 @@ describeStatement returns [std::unique_ptr<cql3::statements::raw::describe_state
         | tK=unreserved_keyword                     { generic_name = tK; } )
                                                     { $stmt = cql3::statements::raw::describe_statement::generic(keyspace, generic_name); }
     )
-    ( K_WITH K_INTERNALS { $stmt->with_internals_details(); } )?
+    ( K_WITH K_INTERNALS (K_AND K_PASSWORDS { with_salted_hashes = true; })?
+        { $stmt->with_internals_details(with_salted_hashes); } )?
```

Comment:
Need to update docs/cql/cql-extensions.md and docs/cql/cqlsh.rst (until we move it somewhere else)

Reply:
Thank you. I didn't notice the first file at all before, but it's definitely a place I should change too. I briefly touched upon `DESCRIBE` in said documents. I'm leaving this conversation as unresolved in case I should elaborate there more or something is still missing